<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="FieldglassSuite" duration-ms="63161" started-at="2022-01-11T10:20:51 IST" finished-at="2022-01-11T10:21:54 IST">
    <groups>
    </groups>
    <test name="TimesheetReportingStatus" duration-ms="63161" started-at="2022-01-11T10:20:51 IST" finished-at="2022-01-11T10:21:54 IST">
      <class name="testCases.TC_001_LoginTC">
        <test-method status="PASS" signature="setUP()[pri:0, instance:testCases.TC_001_LoginTC@3b2da18f]" name="setUP" is-config="true" duration-ms="4121" started-at="2022-01-11T10:20:51 IST" finished-at="2022-01-11T10:20:55 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUP -->
        <test-method status="PASS" signature="test_Login(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC_001_LoginTC@3b2da18f]" name="test_Login" duration-ms="33917" started-at="2022-01-11T10:20:56 IST" data-provider="LoginData" finished-at="2022-01-11T10:21:30 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[C5275209]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Birth@1991]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[1.12.2021]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[31.12.2021]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_Login -->
        <test-method status="FAIL" signature="test_Login(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC_001_LoginTC@3b2da18f]" name="test_Login" duration-ms="22742" started-at="2022-01-11T10:21:30 IST" data-provider="LoginData" finished-at="2022-01-11T10:21:53 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[raushan]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[abcxyz]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[1.12.2021]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[31.12.2021]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at testCases.TC_001_LoginTC.test_Login(TC_001_LoginTC.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_Login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testCases.TC_001_LoginTC@3b2da18f]" name="tearDown" is-config="true" duration-ms="823" started-at="2022-01-11T10:21:53 IST" finished-at="2022-01-11T10:21:54 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC_001_LoginTC -->
    </test> <!-- TimesheetReportingStatus -->
  </suite> <!-- FieldglassSuite -->
</testng-results>
